### ISSUE TITLE
Add timing for test runs

### ISSUE THREAD

**Issue by sgerhardt** [dart-lang/test#397]
It'd be nice to have an option, say '-t', which output the time each test took to run, as well as the total time for all tests.

**Comment by nex3** [comment]
Tests are run in parallel as much as possible, which means that test-by-test timing information won't provide very clean data—how many resources a test gets will be heavily influenced by which other tests happen to be running in parallel. Given that, I don't think I want to include support for measuring it in the test runner itself. However, it's pretty easy to write your own function that would instrument all tests in a suite to provide that information:

```dart
void printTiming() {
  var stopwatch = new Stopwatch();
  setUp(() {
    stopwatch.start();
  });

  tearDown(() {
    stopwatch.stop();
    print("Took ${stopwatch.elapsed}.");
    stopwatch.reset();
  });
}
```

As for timing information across all tests, I think that's better accomplished by running the runner with the `time` command.

**Comment by sgerhardt** [comment]
@nex3 Thanks for the insights. The time command and stopwatch used in this manner will achieve what I'm looking for nicely.
